packages/core/src/constants/element.constant.ts

```diff
@@ -1,8 +1,118 @@
-export const ELEMENT_NAMES = {
-	substation: 'Substation',
-	voltageLevel: 'Voltage Level',
-	bay: 'Bay',
-	ied: 'IED',
-	lDevice: 'Logical Device',
-	lNode: 'Logical Node'
+export const BASE_STANDARD_ATTRIBUTES = ['id', 'desc', 'name'] as const
+
+export const SCD_ELEMENTS = {
+	substation: {
+		element: {
+			tag: 'Substation',
+			name: 'Substation',
```

Comment:
To utilize TS we can think about creating types or interfaces for the SCD_Elements. Thoughts?

Reply:
It already exists and defined here:
`/scd-queries/data-type-templates/types.data-type-templates.d.ts`: `TypeElements`

Reply:
I wrote this part as the source of truth of our plugins, and is seen by TS as a constant so ReadOnly properties. I use this to create our types (so it kind of works the other way around 🙃 )

Reply:
Solved